subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1477,15 +1477,6 @@ static int light_ctrl_srv_start(struct bt_mesh_model *mod)
 		return -EINVAL;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_MESH_MODEL_EXTENSIONS)) {
```

Comment:
This does not seem correct. The description of the PR does not match with the change and the file name where this is changed.